app/presenters/document_presenter.rb

```diff
@@ -76,14 +75,10 @@ def metadata
     metadata.reject { |_k, v| v.blank? }
   end
 
-  def public_updated_at
-    document.public_updated_at.to_datetime.rfc3339
-  end
-
   def change_history
     case document.update_type
     when "major"
-      document.change_history + [{ public_timestamp: public_updated_at, note: document.change_note || "First published." }]
+      document.change_history + [{ public_timestamp: STUB_TIME_STAMP, note: document.change_note || "First published." }]
```

Comment:
Oh right, sorry. Just read your commit message. I think it'd be useful to add a big ugly comment above `STUB_TIME_STAMP` to explain something similar to your commit.